### PR TITLE
Publish samples into azure samples browser

### LINKED ISSUE #8: Publish samples into azure samples browser
Origingal Issue :  https://github.com/Azure-Samples/azure-django-postgres-flexible-aca/pull/85

cc: @jianingwang123

I'll move the code changes from the original repo to this repo. Thank you for your contribution.

I'll also make sure to add you as a participant

### PR DESCRIPTION
**Pull request by kjaymiller**
This pull request publishes the samples into the Azure Samples browser. It includes changes to the necessary files and updates the YAML front-matter schema. Fixes #8.

### CODE REVIEW

**Comment by kjaymiller** [comment]
cc: @jianingwang123

File: {{cookiecutter.__src_folder_name}}/README.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+# {% if cookiecutter.project_host == "appservice" %}
+- azure-app-service
+# {% endif %}
+# {% if "postgres" in cookiecutter.db_resource %}
+- azure-postgresql
+# {% endif %}
+# {% if "cosmos" in cookiecutter.db_resource %}
+- azure-cosmos-db
+# {% endif %}
+- azure-pipelines
```
Comment (pamelafox) [inline {{cookiecutter.__src_folder_name}}/README.md:26]: How are azure-pipelines used?

File: {{cookiecutter.__src_folder_name}}/README.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+urlFragment: azure-{{cookiecutter.project_backend}}-{{cookiecutter.db_resource}}-{{cookiecutter.project_host}}
+name: Deploy {{web_framework}} Application with {{db}} via {{azure_host}} (Python)
```
Comment (pamelafox) [inline {{cookiecutter.__src_folder_name}}/README.md:28]:
```suggestion
name: Deploy {{web_framework}} Application with {{db}} on {{azure_host}} (Python)
```

File: {{cookiecutter.__src_folder_name}}/README.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+description: This project deploys a web application for a space travel agency using {{web_framework}} with Python.
```
Comment (pamelafox) [inline {{cookiecutter.__src_folder_name}}/README.md:29]:
```suggestion
description: This project deploys a web application for a space travel agency using {{web_framework}} with Python, and is set up for easy deployment with the Azure Developer CLI.
```

**Pull request merged**